.sass-lint.yml

```diff
@@ -0,0 +1,103 @@
+#########################
+## Sample Sass Lint File
+#########################
+# Linter Options
+options:
+  # Set the formatter to 'html'
+  formatter: stylish
+# File Options
+files:
+  include: 'src/**/*.scss'
+# Rule Configuration
+rules:
+  # Extends
+  extends-before-mixins: 2
+  extends-before-declarations: 2
+  placeholder-in-extend: 2
+
+  # Mixins
+  mixins-before-declarations:
+    - 2
+    -
+      exclude:
+        - 'respond-at-and-above'
+        - 'respond-between'
+        - 'respond-below'
+        - 'after'
+        - 'before'
+
+  # Line Spacing
+  one-declaration-per-line: 2
+  empty-line-between-blocks: 2
+  single-line-per-selector: 2
+
+  # Disallows
+  no-color-keywords: 2
+  no-color-literals: 0
```

Comment:
@robwierzbowski https://github.com/sasstools/sass-lint/blob/develop/docs/rules/no-color-literals.md

I'm not as familiar with our current Sass practices, but I do like the `no-color-literals` rule because it forces you to use our standard color functions/variables. For example, this is not allowed:

```background: 1px solid #asdf;```

Instead, you would need to do:

```$brand-color: #asdf;
background: 1px solid $brand-color;```